bin/check-redis-list-length.rb

```diff
@@ -63,19 +63,38 @@ class RedisListLengthCheck < Sensu::Plugin::Check::CLI
          description: 'Redis list KEY to check',
          required: true
 
+  option :threshold,
```

Comment:
I think I prefer this to pass in something like `>=` as this adds max flexibility and does not require another conditional.